ext/lib/mgmt/mcumgr/README.md

```diff
@@ -0,0 +1,138 @@
+# mcumgr
+
+This is mcumgr, version 0.0.1
+
+mcumgr is a management library for 32-bit MCUs.   The goal of mcumgr is to
+define a common management infrastructure with pluggable transport and encoding
+components.  In addition, mcumgr provides definitions and handlers for some
+core commands: image management, file system management, and OS managment.
+
+mcumgr is operating system and hardware independent.  It relies on hardware
+porting layers from the operating system it runs on.  Currently, mcumgr runs on
+both the Apache Mynewt and Zephyr operating systems.
+
+## Getting started
+
+For tips on using mcumgr with your particular OS, see the appropriate file from
+the list below:
+
+* README-mynewt.md
+* README-zephyr.md
+
+## Dependencies
+
+To use mcumgr's image management support, your device must be running the
+MCUboot boot loader (https://github.com/runtimeco/mcuboot).  The other mcumgr
+features do not require MCUboot.
+
+## Command line tool
+
+The `mcumgr` command line tool is available at:
+https://github.com/apache/mynewt-mcumgr-cli.  The tool is written in Go, and it
+is installed with the `go get` command:
+
+```
+$ go get github.com/apache/mynewt-mcumgr-cli/mcumgr
```

Comment:
FYI:

Running this command on Ubuntu 16.04 with go version go1.6.2 linux/amd64 results in:

```
sebo@mach:~/mcuboot/boot/zephyr$ go get github.com/apache/mynewt-mcumgr-cli/mcumgr
package github.com/apache/mynewt-mcumgr-cli/mcumgr: cannot download, $GOPATH not set. For more details see: go help gopath

```

Fixing this with 

```
sebo@mach:~/mcuboot/boot/zephyr$ mkdir ~/gocode
sebo@mach:~/mcuboot/boot/zephyr$ export GOPATH=~/gocode

```

results in:

```
sebo@mach:~/mcuboot/boot/zephyr$ go get github.com/apache/mynewt-mcumgr-cli/mcumgr
package context: unrecognized import path "context" (import path does not begin with hostname)

```

Reply:
The `context` go package requires Go >=1.7.  Thanks for the heads up; I will add this to the readme.